README.md

```diff
@@ -54,7 +54,13 @@ If you are not using Cocoapods, Link the Framework from [here](https://github.co
 
 ```javascript
 import SimplSdk from 'react-native-simpl-zero-click-sdk'
-//To check either user is approved or not
+
+// To check eithes user is approved or not (new api)
+ SimplSdk.isUserApproved({merchantId: '<merchant id>', isSandbox: true, phone_number: '<phone_number>', email: '<email>', params: { amount_in_paisa: 1000 }},
```

Comment:
extract the first param as a variable.
`amount_in_paise` and not `amount_in_paisa`